### PR TITLE
Bundling optimization

### PR DESCRIPTION
**Pull request by chad-ramos**
References #497 

@ferventcoder - Being you expressed a desire to work with Cassette, I went ahead and got it working with the current `style.css` file that you are serving in production. 

### Issue
As Cassette went through its build logic, it was running up against a class style that it appeared to be interpreting as UTF16 but had an expectation of UTF32.  Of which, was throwing an out-of-range error.  Long story short, there is a bug inside of Cassette. 

### Fix
 The offending style was `.terminal-chrome__close`.  Removing it has zero effect on the UI.  I am assuming the content type of `\c02715` produces an **X** value, of which you have replaced with `&times;`.  That being said, I removed it from `style.css` and verified the UI was unchanged. 

### Optimization
There were a few low-hanging-fruit things I did in the style sheet.  The commit notes should clearly outline them. 

### Thoughts
Personally, I dislike Cassette.  There are odd constructs enforced to manipulate your desired outcome. Of which to me, speaks to its age.  

* Forcing a full build to view changes seems like a nightmare for development.
* No guarantee you will not run into this bug again.
* Building bundles around none explicit configuration is not intuitive to desired expectations.

Yes, it works but I believe there are better options available today that align a bit more with how the web works.

### Results
| style.css  | Size |
| ------------- | ------------- |
| Current Production  | 62.6kb  |
| Pull Request | 46.9kb  | 

Every kb counts, :smile:

### CODE REVIEW

File: chocolatey/Website/Content/style.css [hunk 1]
```diff
@@ -2523,10 +2513,6 @@ img  { max-width: 100%; }
   box-shadow: 5px 7px 50px rgba(0,0,0,.2);
 }
 
-.terminal-chrome__close {
-  content: '\c02715';
-}
```
Comment (ferventcoder) [inline chocolatey/Website/Content/style.css]: This was the x for the terminal on the main homepage IIRC. No idea if it actually ended up being used though.

Comment (ferventcoder) [inline chocolatey/Website/Content/style.css]: https://chocolatey.org/

Unchecking it seems to have zero effect on removing the "X" though.

File: chocolatey/Website/Content/style.css [hunk 2]
```diff
@@ -3451,7 +3437,7 @@ fieldset.form  {
             background-image: -webkit-linear-gradient(top, #1A375A 0%, #21446D 100%);
             background-image: linear-gradient(top, #1A375A 0%, #21446D 100%);
             border: solid 1px #21446D;
-            box-shadow: inset 0px 0px 1px rgba(255,255,255,1), 1px 1px 1px rgba(0,0,0,0.3);
+            box-shadow: inset 0 0 1px rgba(255,255,255,1), 1px 1px 1px rgba(0,0,0,0.3);
```
Comment (ferventcoder) [inline chocolatey/Website/Content/style.css]: So you remove 0px, but leave the ones with numbers. Just trying to understand the optimization. :D

File: chocolatey/Website/Infrastructure/CassetteConfiguration.cs [hunk 3]
```diff
@@ -31,12 +31,7 @@ public void Configure(BundleCollection bundles)
             // Each immediate sub-directory of ~/Scripts will be combined into its own bundle.
             // This is useful when there are lots of scripts for different areas of the website.
 
-            bundles.AddPerIndividualFile<StylesheetBundle>("Content", new FileSearch
-            {
-                Pattern = "*.css",
-                SearchOption = SearchOption.AllDirectories,
-                Exclude = new Regex("style\\.css$")
-            });
```
Comment (ferventcoder) [inline chocolatey/Website/Infrastructure/CassetteConfiguration.cs]: Please keep all of this except for line 38 (the exclude)

File: chocolatey/Website/Content/style.css (hunk 1 above)
Comment (chad-ramos) [inline chocolatey/Website/Content/style.css]: Yeah...I looked at also and from what I can see had no effect.

File: chocolatey/Website/Views/Shared/_BaseLayout.cshtml [hunk 4]
```diff
@@ -39,7 +36,6 @@
   <meta property="og:image:height" content="700" />
   <meta property="copyright" content="Chocolatey is Copyright 2011 - @DateTime.UtcNow.Year RealDimensions Software, LLC - a US-based company."/> 
   @Bundles.RenderStylesheets()
-  <link href="@Url.Content("~/content/style.css")?update=20161223" rel="stylesheet" />
```
Comment (ferventcoder) [inline chocolatey/Website/Views/Shared/_BaseLayout.cshtml]: 👍

File: chocolatey/Website/Content/style.css (hunk 2 above)
Comment (chad-ramos) [inline chocolatey/Website/Content/style.css]: Unit identifiers are optional.  You will save two characters per of file size.  No performance increase though.

Comment (chad-ramos) [inline chocolatey/Website/Content/style.css]: You need to specify for none zero values though....

**Comment by chad-ramos** [comment]
@ferventcoder Sure...sorry I overlooked the guidelines. Hence me being verbose with the commit history.  I will put it all together with your requested changes later on today.

**Comment by ferventcoder** [comment]
@chad-ramos Much appreciated on verbose. Always good to start from more verbose because it is easier to make it less verbose. Harder to go the other way. 👍

File: chocolatey/Website/Content/style.css (hunk 2 above)
Comment (ferventcoder) [inline chocolatey/Website/Content/style.css]: Awesome, 👍

Comment (ferventcoder) [inline chocolatey/Website/Content/style.css]: As an aside - you know my github review trick when I want to add more comments.

**Comment by chad-ramos** [comment]
@ferventcoder I just murdered my rebase and squash. :angry: New PR coming in a few.

**Comment by ferventcoder** [comment]
Superseded by #504

**Pull request closed without merging**